.eslintrc

```diff
@@ -56,22 +64,32 @@
     "no-inline-comments": 0,
     "no-invalid-this": 0,
     "no-magic-numbers": 0,
+    "no-mixed-spaces-and-tabs": 2,
+    "no-multi-str": 2,
+    "no-multiple-empty-lines": 2,
     "no-negated-condition": 0,
     "no-shadow": 0, // TODO: Change to `1`?
+    "no-spaced-func": 2,
     "no-trailing-spaces": [2, {"skipBlankLines": false}],
     "no-undef": 2,
     "no-underscore-dangle": 0,
     "no-unused-vars": [2, {"vars": "all", "args": "none"}],
+    "no-var": 2,
     "no-warning-comments": 0, // TODO: Change to `1`?
+    "no-with": 2,
     "object-curly-spacing": [2, "never"],
-    "no-var": 2,
+    "one-var": [2, "never"],
+    "operator-linebreak": [2, "after"],
     "prefer-const": 0, // TODO: Change to `1`?
     "prefer-reflect": 0,
     "quotes": [2, "double", "avoid-escape"],
     "semi": [2, "always"],
+    "space-before-blocks": [2, "always"],
     "space-before-function-paren": [2, {"anonymous": "never", "named": "never"}],
+    "space-in-parens": [2, "never"],
     "space-infix-ops": 2,
     "space-unary-ops": 2,
-    "strict": 0
+    "strict": 0,
+    "valid-jsdoc": [0, {"requireReturn": false, "requireParamDescription": false, "requireReturnDescription": false}]
```

Comment:
Disabled this rule for now as it throws about 72 errors and I'm not sure we're using JSDoc.